web/src/app/models/geometry/geometry.ts

```diff
@@ -33,3 +33,14 @@ export enum GeometryType {
   POLYGON = 'POLYGON',
   MULTI_POLYGON = 'MULTI_POLYGON',
 }
+
+/** A label for a given geometry type. Defaults to 'Polygon'. */
+export function geometryTypeLabel(geometryType?: GeometryType): string {
```

Comment:
The label is presentation logic. Can we move this into the component or into a shared service? This will also make i18n easier later.

Reply:
Moved it back to the loi panel header component for now.